src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -183,7 +183,7 @@ public PackageViewModel(DynamoViewModel dynamoViewModel, Package model)
 
             ToggleTypesVisibleInManagerCommand = new DelegateCommand(() => { }, () => true);
             GetLatestVersionCommand = new DelegateCommand(() => { }, () => false);
-            PublishNewPackageVersionCommand = new DelegateCommand(() => ExecuteWithTou(PublishNewPackageVersion), () => CanPublish);
+            PublishNewPackageVersionCommand = new DelegateCommand(() => ExecuteWithTou(PublishNewPackageVersion), CanDeprecate);
```

Comment:
It might be confusing to use as it is and just rename to `IsOwner()`.

Reply:
Right? I was a bit iffy on using it like that. It serves both needs, I will just rename it to better show the purpose of the method.

Reply:
Done!